File: Exercicio-08.java
import java.util.Scanner;
import java.text.DecimalFormat; 

public class Main
{
  public static void main (String[]args)
  {
    Scanner sc = new Scanner (System.in);
    DecimalFormat df = new DecimalFormat("#,###.00");
    double value, rate, income, total;;

    System.out.printf("Digite o valor do depósito: ");
    value = sc.nextDouble();
  
    System.out.printf("Digite a taxa de juros (Valor em %%): ");
    rate = sc.nextDouble();
      
    income = value * rate / 100;
    total = value + income;
  
    System.out.printf("Valor Depositado: " + df.format(value) + "\n");
    System.out.printf("Valor de Rendimento: " + df.format(income) + "\n");
    System.out.printf("Valor Total: " + df.format(total) + "\n");
  }
}

File: Exercicio-08.c
#include <stdio.h>

int main(void) {

  float value, rate, income, final;

  printf("Digite o valor do depósito: ");
  scanf("%f", &value);
  
  printf("Digite a taxa de juros (Valor em %%): ");
  scanf("%f", &rate);
  
  income = value * rate / 100;
  final = value + income;
  
  printf("Valor Depositado: %.2f\n", value);
  printf("Valor de Rendimento: %.2f\n", income);
  printf("Valor Total: %.2f\n", final);
  
  return 0;
}
